lib/web_ui/lib/src/engine/skwasm/skwasm_impl/filters.dart

```diff
@@ -60,7 +60,7 @@ abstract class SkwasmImageFilter implements SceneImageFilter {
   ///
   /// The handle is deleted immediately after [borrow] returns. The [borrow]
   /// function must not store the handle to avoid dangling pointer bugs.
-  void withRawImageFilter(ImageFilterHandleBorrow borrow);
+  void withRawImageFilter(ImageFilterHandleBorrow borrow, {ui.TileMode defaultMode = ui.TileMode.clamp});
```

Comment:
Two things: can you actually call this `defaultTileMode` to be a bit more clear, and secondly can you modify the doc comment to mention this parameter and what it does?

Reply:
I went with defaultBlurTileMode since it only applies to blurs...